teuthology/provision/fog.py

```diff
@@ -278,7 +278,11 @@ def _wait_for_ready(self):
                     NoValidConnectionsError,
                     MaxWhileTries,
                     EOFError,
-                ):
+                ) as e:
+                    # log this, because otherwise lots of failures just keep retrying without
+                    # any notification (like, say, a mismatched host key in ~/.ssh/known_hosts, or
+                    # something)
+                    log.exception(e)
```

Comment:
```suggestion
                    log.error(e)
```
probably don't want to log the full tb _and_ the exception

Reply:
yes, true.  I don't even think I want error; I think warning is more apropos.  (there are expected timeouts while the host comes up.)